MXLPersonalArmory/IPC/Messaging/Messages/Models/Stats.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MXLPersonalArmory.IPC.Messaging.Messages.Models
+{
+    class Stats

```

Comment:
Stats implies multiple objects - rename to Stat.
Don't forget the file name as well.

Reply:
Fixed.